src/components/ImageView/RegionView/LineSegmentRegionComponent.tsx

```diff
@@ -277,9 +277,11 @@ export class LineSegmentRegionComponent extends React.Component<LineSegmentRegio
         let anchors = null;
         let newAnchor = null;
         let pointArray: Array<number>;
-        // trigger re-render when exporting images
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        // trigger re-render when exporting images and changing devicePixelRatio (switching monitor)
+        /* eslint-disable @typescript-eslint/no-unused-vars */
         const imageRatio = AppStore.Instance.imageRatio;
```

Comment:
Since `pixelRatio` changes with `imageRatio`, we could remove this line.

Reply:
this imageRatio is used later, so I move it out.

Reply:
ok.